apps/files_trashbin/lib/UserMigration/TrashbinMigrator.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2022 Côme Chilliet <come.chilliet@nextcloud.com>
+ *
+ * @author Côme Chilliet <come.chilliet@nextcloud.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Files_Trashbin\UserMigration;
+
+use OCA\Files_Trashbin\AppInfo\Application;
+use OCP\Files\IRootFolder;
+use OCP\Files\NotFoundException;
+use OCP\IUser;
+use OCP\UserMigration\IExportDestination;
+use OCP\UserMigration\IImportSource;
+use OCP\UserMigration\IMigrator;
+use OCP\UserMigration\TMigratorBasicVersionHandling;
+use OCP\UserMigration\UserMigrationException;
+use Symfony\Component\Console\Output\OutputInterface;
+use OCP\IDBConnection;
+
+class TrashbinMigrator implements IMigrator {
+
+	use TMigratorBasicVersionHandling;
+
+	protected const PATH_FILES_FOLDER = Application::APP_ID.'/files';
+	protected const PATH_LOCATIONS_FILE = Application::APP_ID.'/locations.json';
+
+	protected IRootFolder $root;
+
+	protected IDBConnection $dbc;
+
+	public function __construct(
+		IRootFolder $rootFolder,
+		IDBConnection $dbc
+	) {
+		$this->root = $rootFolder;
+		$this->dbc = $dbc;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function export(IUser $user, IExportDestination $exportDestination, OutputInterface $output): void {
+		$output->writeln('Exporting trashbin into ' . Application::APP_ID . '…');
+
+		$uid = $user->getUID();
+
+		try {
+			$trashbinFolder = $this->root->get('/'.$uid.'/files_trashbin');
+			$output->writeln("Exporting trashbin…");
```

Comment:
```suggestion
			$output->writeln("Exporting trashbin files…");
```
Match the wording of the importing message